okhttp/src/main/java/okhttp3/internal/ws/WebSocketWriter.java

```diff
@@ -212,6 +226,13 @@ void writeMessageFrame(int formatOpcode, long byteCount, boolean isFirstFrame,
     sink.emit();
   }
 
+  @Override
+  public void close() throws IOException {
+    if (messageDeflater != null) {
+      messageDeflater.close();
```

Comment:
```kotlin
messageDeflater?.close()
```
[Ah, that’s Java](https://66.media.tumblr.com/5ea028443f6ad88bf6af6eb24f1294be/tumblr_mh2htyUWqO1rzupqxo1_500.png).